templates/adduser.html

```diff
@@ -1,43 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-    <meta charset="UTF-8">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="/static/formstyle.css"> <!-- Link to CSS -->
+    <title>Add New Admin/User</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Reset Password</title>
-    <link rel="stylesheet" href="/static/style.css"> <!-- Link to CSS -->
 </head>
 
 <body>
     <div class="centertextleft">
-        <h1 class="h1smaller">Reset Your Password</h1>
-        <p>Please enter your new password below.</p>
+        <h1 class="h1smaller">Add New Admin/User</h1>
+        <p>Enter the required details below:</p>
     </div>
 
-    <!-- Reset Password Form -->
-    <form action="/open-account" method="POST">
-        <!-- Hidden field for token -->
-        <label for="Name">Name</label>
-        <input type="name" id="name" name="name" required>
-        <label for="email">Email</label>
-        <input type="email" id="email" name="email" required>
+    <form action="/add-user" method="POST">
+        <!-- Admin First Name -->
+        <label for="fname">First Name (Admin)</label>
+        <input type="text" id="fname" name="fname" value="John" required placeholder="Enter first name">

```

Comment:
We probably don't need a placeholder value of "John" since their actual name will be typed in